<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="FirstTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EventButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Действие</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SecondTextEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
